src/codes/clj/contest/submission_runner/controllers/submission.clj

```diff
@@ -0,0 +1,5 @@
+(ns codes.clj.contest.submission-runner.controllers.submission)
+
+(defn submit-code-execution!
+  [submission]
+  (random-uuid))
```

Comment:
If you are generating this just to insert into the database, you can delegate the UUID generation to postgres.

Reply:
still not 100% clear what we are going to save on postgres 
id, code, code-hash, language and test-cases as string?

Reply:
good question, I think I would store information to be possible to trace back the author, contest and the submission.

I would have an table to the contests (where the base challenge, and test inputs (and demo test inputs) would be stored, another for the authors, another for the submission where you would have the challenge id, author id, and the submission itself, to not store to much data in the database I would keep only the latest submission for each challenge and author, so if I submit and answer for a challenge we update the submission table by author-id and challenge-id with the new code, code-hash and updated date

After I would  have a  table with the execution results where you could store submission-id, code-hash and the result.

We can whiteboard this later if you want so.

Reply:
>We can whiteboard this later if you want so.

I would love whiteboard this